document-generation/generate-document.test.ts

```diff
@@ -34,6 +53,10 @@ describe("Successful generate-document handler", () => {
       text-align: center;
       margin-top: 0;
     }`;
+    const dowHeaders = {
+      "Content-Type": "application/pdf",
+      "Content-Disposition": `attachment; filename=DescriptionOfWork.pdf`,
```

Comment:
```suggestion
      "Content-Disposition": `attachment; filename=IGCE.xlsx`,
```

Reply:
After a second look, this is the correct `filename` since that is the header for the DoW. The `filename` for the IGCE doc is in the `igce-document.ts` and is `filename=IndependentGovernmentCostEstimate.xlsx`.